api/forgot_test.go

```diff
@@ -119,14 +125,14 @@ func TestForgotResponds(t *testing.T) {
 			// without shortKey
 			method:   "POST",
 			url:      "/send/forgot/clinic@myemail.com",
-			respCode: 200,
+			respCode: 403,
```

Comment:
can you please update the comments that say everytung is ok and it returns a 200 ;-) 
same for a couple of below comments